azurerm/internal/services/containers/resource_arm_kubernetes_cluster.go

```diff
@@ -120,6 +120,42 @@ func resourceArmKubernetesCluster() *schema.Resource {
 				},
 			},
 
+			"identity_profile": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Computed: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"kubelet_identity": {
+							Type:     schema.TypeList,
+							Optional: true,
+							Computed: true,
+							MaxItems: 1,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"client_id": {
+										Type:     schema.TypeString,
+										Optional: true,
+										Computed: true,
+									},
+									"object_id": {
+										Type:     schema.TypeString,
+										Optional: true,
+										Computed: true,
+									},
+									"resource_id": {
```

Comment:
we avoid the phrase `Resource ID` because it's ambigious between Azure and Terraform - could we instead make this `user_assigned_identity_id`?